artman/cli/main.py

```diff
@@ -113,6 +111,14 @@ def parse_args(*args):
              '/path/to/config.yaml:config_section_A|config_section_B. '
              'This is an advanced feature; for most cases, use `--api`.',
     )
+    api.add_argument('--batch',
```

Comment:
How does this interact with publishers? Are the publishing steps done separately or is there an attempt to amalgamate them all into a single publish? (I believe they should remain distinct steps.)